charts/prometheus-node-exporter/README.md

```diff
@@ -41,6 +41,17 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### 3.x to 4.x
+
+Starting from version 4.0.0, the `node exporter` chart is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therefore you have to delete the deployment before you upgrade.
+
+```console
+kubectl delete daemonset -l app=prometheus-node-exporter
```

Comment:
@monotek looks like the suggestion here is to delete the daemonset now, am I missing anything?